srcs/Client.cpp

```diff
@@ -342,6 +344,42 @@ void Client::setConfig(ConfigGroup &group)
 	}
 }
 
+int	Client::parseBody(std::string &tmp, size_t pos)
+{
+	if (this->_content_length_left != EMPTY_CONTENT_LENGTH)
+	{
+		if (tmp.length() < this->_content_length_left)
```

Comment:
이 부분의 역할이 `현재 가지고 있는 line(tmp)의 길이`가 `Content-Length의 남은 길이`보다 길다면 초과분을 지워주는 게 맞을까요?
그러면 꺽쇠 방향을 고쳐줘야 할 것 같습니다. 혹시 제가 잘못 이해했다면 코멘트 부탁드려요!
```suggestion
		if (tmp.length() > this->_content_length_left)
```

Reply:
맞습니다 ㅠㅜ 제가 햇갈리고 막 코드를 갈겨놨군요 ㅠㅜ